cluster/charts/crossplane/values.yaml.tmpl

```diff
@@ -31,4 +31,7 @@ clusterStacks:
     deploy: false
 
 hostedConfig:
-  enabled: false
\ No newline at end of file
+  enabled: false
+
+templateStacks:
+  enabled: false
```

Comment:
As it doesn't affect any if the existing stuff, I suggest we enable it by default.

Reply:
I disagree with this. It does affect existing stuff since the stack definition controller (and stack configuration controller) run in the same process as the stack manager, and will kill the stack manager if they encounter a fatal error.

I agree with the general sentiment that we'll want to enable it by default soon though. I imagine us doing that once we've hardened it a bit more.